Framework/CurveFitting/test/FuncMinimizers/FABADAMinimizerTest.h

```diff
@@ -190,23 +190,23 @@ class FABADAMinimizerTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(PDF->getNumberHistograms(), nParams + 1);
     TS_ASSERT_EQUALS(PDF->x(0).size(), 21);
     TS_ASSERT_EQUALS(PDF->y(0).size(), 20);
-    TS_ASSERT_DELTA(PDF->y(0)[7], 0.41, 0.01);
-    TS_ASSERT_DELTA(PDF->y(1)[8], 5.24, 0.01);
-    TS_ASSERT_DELTA(PDF->y(2)[0], 0.44, 0.01);
+    TS_ASSERT_DELTA(PDF->y(0)[7], 0.41, 0.3);
+    TS_ASSERT_DELTA(PDF->y(1)[8], 5.24, 1.0);
+    TS_ASSERT_DELTA(PDF->y(2)[0], 0.44, 0.3);
```

Comment:
It is not a deterministic algorithm, therefore the code should not find the same results as before.